VERSION

```diff
@@ -0,0 +1 @@
+v1.0.0
```

Comment:
cc: @lvirbalas

```suggestion
v0.1.0
```
This is partly a NIT, but I am just thinking - should we start with v0.1.0 for now.
This should cover us up if we make any accident. E.g. make a tag or something. Everything up to v1.0.0 is not supposed to be stable so it is more forgiving :)